core/transaction_pool/clock.hpp

```diff
@@ -12,17 +12,19 @@ namespace kagome::transaction_pool {
 
   /**
    * An interface for a clock
+   * @tparam clock type is an underlying clock type, such as std::steady_clock
    */
+   template <typename ClockType>
```

Comment:
add default argument, e.g. 
```
typename ClockType = std::chrono::steady_clock;
```

Reply:
Are you sure? I don't see any clock that fits to be default, they're all substantially different and for it not to be confusing I prefer to keep declaring the clock type explicit.

Reply:
yes, unless you have any specific requirements for the clock, steady clock are good enough to be used as default